### PR TITLE
docs: Fix some typos in helpers/html_helper.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Fix some typos in helpers/html_helper.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/helpers/html_helper.rst [hunk 1]
```diff
@@ -253,52 +251,67 @@ The following functions are available:
...
+    :param  string $kind:        The kind of timed track
+    :param  string $srcLanguage: The language of the timed track
+    :param  string $label:       A user-readable title for the timed track
+    :returns:                    An HTML track element
+    :rtype: string
+
+    Generates a track element to specify timed tracks. The tracks are
+    formatted in WebVTT format. Example:
+
+    .. literalinclude:: html_helper/019.php
```
Comment (kenjis) [inline user_guide_src/source/helpers/html_helper.rst:288]: These differences make it difficult to review.
I don't know why `track()` location was changed.
Please create one commit for changing the position.

Also, there are too many diffs in one commit. 
Many fixes are corrections of errors, and it is too large to be called typos.
Please create a commit with a commit log of what you changed for every single meaningful fix.

File: user_guide_src/source/helpers/html_helper.rst [hunk 2]
```diff
@@ -28,37 +27,36 @@ The following functions are available:
...
 
     .. literalinclude:: html_helper/002.php
 
-    There is an optional second parameter that is a true/false value that
-    specifics if the *src* should have the page specified by
-    ``$config['indexPage']`` added to the address it creates.
-    Presumably, this would be if you were using a media controller:
+    There is an optional second parameter, a true/false value, that
+    specifies if the *src* should have ``$config['indexPage']`` added to the address it creates.
+    Presumably, this would be true if you were using a media controller???:
```
Comment (ddevsr) [inline user_guide_src/source/helpers/html_helper.rst:43]:
```suggestion
    Presumably, this would be true if you were using a media controller:
```

**Review by paulbalandan — APPROVED** [review]

**Review by kenjis — CHANGES_REQUESTED** [review]
Could you please re-create this PR so that each meaningful change is committed as one git commit? Because I cannot review this PR.

E.g., 
a commit to move a paragraph. 
a commit to replace tags with elements.
a commit to fix one function description.

**Comment by kenjis** [comment]
Closed by #9153, #9161, #9162, #9163, #9164

**Pull request closed without merging**